.CONTRIBUTING.md

```diff
@@ -0,0 +1,316 @@
+
+
+## Edit Online with GitHub (Simple Changes)
+
+For simple contributions like fixing typos or making small text changes, you can use GitHub's online editor.
+
+### Making Your Changes
+
+1. Navigate to the file you want to edit on GitHub.
+2. Click the pencil icon (✏️) to edit the file.
+3. Make your changes in the online editor.
+4. Follow the [PaperMoon Style Guide](https://github.com/papermoonio/documentation-style-guide).
+
+### Creating Your Pull Request
+
+1. Scroll to **Propose changes** at the bottom of the editor.
+2. Add a descriptive commit message explaining your changes.
+3. Click **Propose changes**. GitHub will automatically:
+   - Fork the repository to your account.
+   - Create a branch with your changes.
+   - Open a pull request.
+      - Make sure to check "Allow edits from maintainers"
+
+## Fork and Edit Locally
+
+For more complex contributions, which involve rendering (e.g., adding a `.Nav` section, code snippets, etc) or when you want to preview changes locally.
+
+### Initial Setup
+
+Before making contributions, you need to set up the proper directory structure for local development:
+
+1. Clone the MkDocs repository:
+
+   > **polkadot-mkdocs**: Contains the "engine" (MkDocs configuration, theme, styling)
+
+   ```bash
+   git clone https://github.com/papermoonio/polkadot-mkdocs
+   cd polkadot-mkdocs
+   ```
+
+2. Fork and clone the docs repository inside polkadot-mkdocs:
+
+   > **polkadot-docs**: Contains the "content" (documentation pages, tutorials, images)
+
+   ```bash
+   # Fork polkadot-docs on GitHub first, then:
+   git clone https://github.com/your-username/polkadot-docs.git
+   cd polkadot-docs
+   git checkout -b your-feature-branch
+   ```
+
+   Your directory structure should now look like:
+   ```
+   polkadot-mkdocs/
+   ├── polkadot-docs/          # Your forked repository
+   ├── material-overrides/
+   └── mkdocs.yml
+   ```
+
+### Making Changes
+
+1. Make your changes:
+
+   - Follow the [PaperMoon Style Guide](https://github.com/papermoonio/documentation-style-guide).
+   - Test your changes locally (see [Run Polkadot Docs Locally](../README.md#run-polkadot-docs-locally)).
+
+2. Update AI documentation:
+
+   After making content changes, regenerate the AI-ready documentation files. 
+   
+   Set up your Python environment following the [Set Up Python Environment](./README.md#set-up-python-environment) section of the README, then run:
+   ```bash
+   python3 scripts/generate_llms.py
+   ```
+   **Note**: Run this script on every PR once all changes are committed.
+
+3. Create pull request:
+
+   - Push your branch and create a pull request.
+   - Use the PR template to indicate your review preference.
+
+## Fix or Improve Existing Content
+
+Making changes to existing pages is the simplest contribution:
+
+1. **Edit the content**: Make your improvements directly to the existing markdown file.
+2. **Follow page requirements**: Ensure your changes maintain proper formatting.
```

Comment:
If I jump down to this section from the TOC, follow page requirements is kind of vague, like what are the page requirements?

Reply:
Agreed. Updated.